katello/katello-change-hostname

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/ruby
+
+require "socket"
+require 'highline/import'
+
+STDOUT.puts "Starting the hostname change..."
+
+STDOUT.puts("\nA hostname change requires a manual change in Satellite. \n
+            Please log into UI and rename the default capsule (under infrastructure->capsules) 
+            to the new hostname using both the name and url fields.")
+
+unless HighLine.agree('Have you made this necessary change and want to proceed with the hostname change? [yes/no]')
+  STDOUT.puts "Hostname change aborted, no changes have been made to your system"
+  exit
+end
```

Comment:
since it appears that the changes below cause a new CA to be generated, should we also warn that all clients will need their bootstrap rpm upgraded/re-installed?